.gitignore

```diff
@@ -30,3 +30,13 @@ proguard/
 
 # Android Studio captures folder
 captures/
+
+.gitignore~
+UiPharmecy/.idea/*
+
+assets/server_source/.idea/
+lint.xml
+
+.metadata/
+
+.settings/
```

Comment:
add .gitignore in .gitignore file. then stage, commit and push. DO NOT ADD FIRST AND THEN EDIT .gitignore. Please remember.